networking/libsnnet/vnic_test.go

```diff
@@ -184,6 +184,52 @@ func TestVnicContainer_GetDevice(t *testing.T) {
 	performVnicOps(true, assert, vnic)
 }
 
+//Test ability to attach to an existing VNIC
+//
+//Tests the ability to attach to an existing
+//VNIC and perform all VNIC operations on it
+//
+//Test is expected to pass
+func TestVnic_GetDeviceByName(t *testing.T) {
+	assert := assert.New(t)
+	vnic1, _ := NewVnic("testvnic")
+	vnic1.LinkName = "testiface"
+
+	assert.Nil(vnic1.Create())
+	vnic, _ := NewVnic("testvnic")
+
+	assert.Nil(vnic.GetDeviceByName("testiface"))
+	assert.NotEqual(vnic.InterfaceName(), "")
+	assert.Equal(vnic.InterfaceName(), vnic1.InterfaceName())
+	assert.NotEqual(vnic1.PeerName(), "")
```

Comment:
It works because of https://github.com/01org/ciao/blob/master/networking/libsnnet/vnic.go#L66
I forgot about that. So this is ok